_posts/molecules/2019-02-18-collapsible_text.html

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Collapsible text
+category: molecules
+description: Use the 'collapsible_text' partial when you want to show a subset of text
+             by default and allow users to collapse it via a 'Show more' link.
+---
+
+// = render partial: 'webui/webui/collapsible_text', locals: { text: 'Text goes here', threshold: 100 }
+- text = "Text\n goes\n here"
+- threshold = 1
+.obs-collapsible-textbox
+  - if text.length > threshold
+    .obs-collapsible-text.obs-collapsed
+      = simple_format(text, class: 'm-0 p-0')
```

Comment:
`simple_format` is not working in Jekyll. It can be a comment like what we have with the `render partial` above.